repondeur/zam_repondeur/views/lectures.py

```diff
@@ -0,0 +1,185 @@
+import os
+import re
+from dataclasses import dataclass
+from tempfile import NamedTemporaryFile
+from typing import Any, Generator, List
+
+from pyramid.httpexceptions import HTTPBadRequest, HTTPFound
+from pyramid.request import Request
+from pyramid.response import FileResponse, Response
+from pyramid.view import view_config, view_defaults
+
+from zam_aspirateur.amendements.models import Amendement
+from zam_aspirateur.amendements.fetch import fetch_and_parse_all, NotFound
+from zam_aspirateur.amendements.writer import write_csv, write_xlsx
+from zam_aspirateur.__main__ import process_amendements
+
+
+CHAMBRES = {"assemblee": "Assemblée nationale", "senat": "Sénat"}
+
+SESSIONS = ["2017-2018"]
```

Comment:
Do you think it's the responsibility of the aspirateur or of the repondeur to deal with dynamic sessions and text numbers? I think we should retrieve that data from http://www.senat.fr/dossiers-legislatifs/depots/depots.html except if there are others (easier) file formats to deal with.

Reply:
Note: that's something that might be useful for the visionneuse too in order to jump from one project to another.

Reply:
Yes, we need to fetch (and probably cache) those lists. Not sure yet how it fits in the overall architecture. Let's talk about this together!